Add tr locale

iozozturk opened the pull request:
Adds Turkish locale

jama5262 commented:
Hi, @iozozturk thanks for your contribution. I will request some few changes

1. Pull requests cannot be accepted to the `master` branch, please send PR to `develop` branch

2. Please write tests for the `tr` locale. Refer to this [PR](https://github.com/jama5262/jiffy/pull/43/files)

3. Please run this in the command line to check for formatting
`
dartfmt -n --set-exit-if-changed lib test && dartanalyzer lib test example
`
If there are formatting issues, please run the following 
`
dartfmt -w lib test
`
This is used to format the package so that the Travis tests pass

iozozturk commented:
Added tr locale tests and ran dartfmt for my changes. Also rebased my master onto develop and targeted `develop` branch. Can you please check @jama5262 ?
